app/models/devise_token_auth/concerns/tokens_serialization.rb

```diff
@@ -0,0 +1,18 @@
+module DeviseTokenAuth::TokensSerialization
+  # Serialization hash to json
+  def self.dump(object)
+    object.nil? ? '{}' : object.each_value(&:compact!).to_json
```

Comment:
I noticed that value of `last_token` very often equals nil (it seems batch) while `updated_at` is still present and updated. So delete (`&:compact!`) all keys with nil value.